src/schema/objects/metadata.yaml

```
@@ -2825,6 +2825,57 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Distance between the participant's eye and the screen. If no screen was used, use `n/a`.
+  anyOf:
+    - type: number
+      unit: cm
+    - type: string
+      enum:
+        - n/a
+ScreenRefreshRate:
+  name: ScreenRefreshRate
+  display_name: Screen Refresh Rate
+  description: |
+    Refresh rate of the screen (when used), in Hertz (equivalent to frames per second, "FPS").
+  type: number
+  unit: Hz
+ScreenResolution:
+  name: ScreenResolution
+  display_name: Screen Resolution
+  description: |
+    Screen resolution in pixel
+    (for example `[1920, 1200]` for a screen of 1920-length by 1080-height pixels),
```

```suggestion
    (for example `[1920, 1200]` for a screen of 1920-width by 1080-height pixels),
```

not a native speaker here but length is IMHO ambiguous and height typically goes with width

---

good catch